cups/translate-attrs_test.go

```diff
@@ -207,13 +207,58 @@ func TestConvertMarkers(t *testing.T) {
 	}
 
 	pt = map[string][]string{
-		attrMarkerNames:  []string{"black", "color", "rainbow", "zebra", "pony"},
-		attrMarkerTypes:  []string{"toner", "toner", "ink", "staples", "water"},
+		attrMarkerNames:  []string{"black", "color"},
+		attrMarkerTypes:  []string{"toner", "toner", "ink"},
+		attrMarkerLevels: []string{"10", "11", "12"},
+	}
+	m, ms = convertMarkers(pt)
+	if m != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+	if ms != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+
+	pt = map[string][]string{
+		attrMarkerNames:  []string{"black", "color", "rainbow"},
+		attrMarkerTypes:  []string{"toner", "toner"},
+		attrMarkerLevels: []string{"10", "11", "12"},
+	}
+	m, ms = convertMarkers(pt)
+	if m != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+	if ms != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+
+	pt = map[string][]string{
```

Comment:
I think this is identical to the previous test, lines 224 through 237. Did you mean to make attrMarkerTypes longer and attrMarkerLevels shorter?